package.json

```
@@ -41,10 +43,17 @@
     "@angular-eslint/template-parser": "17.2.1",
     "@angular/cli": "17.2.1",
     "@angular/compiler-cli": "17.2.0",
+    "@storybook/addon-essentials": "^7.6.17",
```

remove carrots
```suggestion
    "@storybook/addon-essentials": "7.6.17",
```

for all packages